tests/apollo/test_skvbc_reconfiguration.py

```diff
@@ -128,7 +128,7 @@ async def test_wedge_command_and_specific_replica_info(self, bft_network):
         with trio.fail_after(seconds=90):
             done = False
             while done is False:
-                msg = skvbc.get_have_you_stopped_req()
+                msg = skvbc.get_have_you_stopped_req(1)
```

Comment:
Can you use a key-word parameter here? Otherwise `1` looks like some magic number here 😄

Reply:
done